packages/eslint-config-airbnb-base/rules/es6.js

```diff
@@ -10,9 +10,6 @@ module.exports = {
       'objectLiteralDuplicateProperties': false
```

Comment:
It's documented [here](http://eslint.org/docs/user-guide/migrating-to-2.0.0#language-options). Still useful :-)